optimizely/project_config.py

```diff
@@ -85,6 +91,13 @@ def __init__(self, datafile: str | bytes, logger: Logger, error_handler: Any):
         self.host_for_odp: Optional[str] = None
         self.all_segments: list[str] = []
 
+        region_value = config.get('region')
+        self.region: Region
+        if region_value == Region.EU.value:
+            self.region = Region.EU
+        else:
+            self.region = Region.US
```

Comment:
Another area we can get benefits with string type. No need to change the code when we add countries.